be/src/exprs/binary_functions.cpp

```diff
@@ -0,0 +1,212 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "exprs/binary_functions.h"
+
+#include "column/binary_column.h"
+#include "column/column_builder.h"
+#include "column/column_helper.h"
+#include "column/column_viewer.h"
+#include "column/nullable_column.h"
+#include "exprs/base64.h"
+#include "exprs/encryption_functions.h"
+#include "exprs/string_functions.h"
+#include "gutil/strings/escaping.h"
+
+namespace starrocks {
+
+namespace {
+template <bool is_throw_exception>
+static StatusOr<ColumnPtr> _from_hex(const ColumnPtr& src_column) {
```

Comment:
We have already supported hex_decode_binary/base64_decode_binary, can we reuse the same implementation?

Reply:
updated. use `base64_decode_binary ` when it is `to_binary(varchar, 'encode64')`.

but `hex_decode_binary ` is not used yet, because `hex_decode_binary ` now converts binary to digitals string. but
to_binary(varchar, 'hex') converts 'abab' to x'abab' instead.

Reply:
My mistake. `to_binary`/`from_binary` now all re-use all existed functions for now.